Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -155,6 +155,16 @@ def euclidian_modulus(a, b):
     c = (-a) % bp
     return c if c == 0 else bp - c
 
+def newArray(initValue, *dims):
+    b = initValue
+    dimslist = list(dims)
+    for i in reversed(dimslist):
+        tmplist = []
+        for _ in range(i):
+            tmplist.append(b)
+        b = tmplist
```

Comment:
```suggestion
    for i in reversed(list(dims)):
        b = [copy.deepcopy(b) for _ in range(i)]
```
This doesn't do what we though it did.